### ISSUE TITLE
[Suggestion] Optionally show descriptions on countdowns

### ISSUE THREAD

**Issue by wkrueger** [BigWigsMods/BigWigs#1201]
## Describe the problem

 - When I set countdowns I usually get confused on what the countdown is for. An additional clue would be nice.

 - I've commonly seen groups creating WAs to show something like "TRAPS 3 2 1" which is mostly the same as a countdown with description.

I've changed a line in Countdown.lua and got something like this.

![image](https://user-images.githubusercontent.com/2209181/163469874-0c25016f-deae-488b-abc7-989057cd1e15.png)

![image](https://user-images.githubusercontent.com/2209181/163470065-4e2bba15-5835-415e-ac66-24d3185ac5e1.png)

If added as an optional setting this would not bother existing users and maybe allow some formatting for it to not look so ugly as in my print.

Thanks

**Comment by funkydude** [comment]
Putting the entire text in the countdown would be ridiculous. There's already too much information on a players screen as it is.

If you're enabling that many countdowns that you don't know what each of them are, you should be assigning different countdown voices to them. There's already several voices and you can install more such as the overwatch voices.

**Comment by wkrueger** [comment]
I agree that putting the description with the same font size as the countdown is very ugly. That was mostly a quick fix. A small placeholder for the description would be more adequate.

Sometimes I've even used countdowns like that without a sound alarm. Visual vs. sound mostly a matter of preference.

Well, I'll prob try patching that as a separate addon and see if I can make something visually acceptable.

**Comment by funkydude** [comment]
A method that might improve this whilst not being too intrusive would be to have the spell/ability icon next to the text, the same way messages do. The problem would be the size and space it takes, but it could work in conjunction with a transparency slider that would default to 0 (off).
